src/status_im/ipfs/core.cljs

```diff
@@ -4,8 +4,8 @@
 
 ;; we currently use an ipfs gateway but this detail is not relevant
 ;; outside of this namespace
-(def ^:const ipfs-add-url "https://ipfs.infura.io:5001/api/v0/add")
-(def ^:const ipfs-cat-url "https://ipfs.infura.io/ipfs/")
+(def ipfs-add-url "https://ipfs.infura.io:5001/api/v0/add?cid-version=1")
```

Comment:
just wondering why don't you like const?

Reply:
I put them back it was a pita when I had to experiment with urls